features/promotion/managing_catalog_promotions/creating_catalog_promotion.feature

```diff
@@ -46,3 +46,30 @@ Feature: Creating a catalog promotion
         And it should have "winter_sale" code and "Winter sale" name
         And the catalog promotion "Winter sale" should be available in channel "United States"
         And this catalog promotion should be usable
+
+    @api @ui
+    Scenario: Create a catalog promotion with rule
+        Given the store has a "T-Shirt" product
+        When I want to create a new catalog promotion
+        And I specify its code as "winter_sale"
+        And I name it "Winter sale"
+        And I specify its label as "Winter -50%" in "English (United States)"
+        And I describe it as "This promotion gives a 50% discount on all products" in "English (United States)"
+        And I specify rule with configuration for "T-Shirt" product
```

Comment:
For now, we do not have a rule for products, we have only for variants and there is already a scenario for that